### DISCUSSION TITLE
multi-column layout ?

### DISCUSSION THREAD

**Discussion by PierreHentges** [forever-jekyll/forever-jekyll#11] [General]
I love this theme, esp. how simple and clean it is. 

Something that's not clear to me though: can we build a layout structure with several columns or a grid? or is it only possible to have a front page with a single column of blog post truncated teasers?

**Comment by forever-jekyll** [comment]
Hi there @PierreHentges 
First of all sorry for the delayed response.
Sadly, Forever Jekyll is kind of hard-coded to render the single column with teases layout and hence there is no easy way to restructure it to a multi-column or grid view.
Thanks for your encouraging words and sorry again for the delay in responding to your question.
